### PR TITLE
Fix/add check keypair endpoint

### PR DESCRIPTION
**Pull request by saltz**

### CODE REVIEW

File: src/__tests__/cryptoKey.test.ts [hunk 1]
```diff
@@ -35,6 +35,26 @@ describe("Generating key pair", () => {
     });
 });
 
+describe("Generating key pair from seperate keys", () => {
+    it("Should succeed with valid public and pirvate key strings", () => {
+        const keyPair = cryptoKeyUtil.generateKeyPair();
+
+        const result = cryptoKeyUtil.verifyKeyPair(keyPair.getPublic(true, "hex") as string, keyPair.getPrivate("hex"));
```
Comment (Svenmarim) [inline src/__tests__/cryptoKey.test.ts]: Make a const for "hex", so you don't need to change it on 2 places when there is need to change it

File: src/__tests__/cryptoKey.test.ts [hunk 2]
```diff
@@ -71,6 +91,7 @@ describe("Verifying signature", () => {
         try {
             cryptoKeyUtil.verifySignature(secondKeyPair.getPublic().encode("hex", true) as string,
```
Comment (Svenmarim) [inline src/__tests__/cryptoKey.test.ts]: And use the same const here

File: src/cryptoKeyUtil.ts [hunk 3]
```diff
@@ -19,13 +19,24 @@ export class CryptoKeyUtil {
...
-        }
+    /**
+     * Verify if the key combinration is a valid keypair.
+     * @param publicKey 
+     * @param privateKey 
+     * @returns {IKeyPair} EC key pair.
+     */
+    public verifyKeyPair(publicKey: string, privateKey: string): IKeyPair {
+        const keyPair = this.ec.keyPair({
+            priv: Buffer.from(privateKey, "hex"),
```
Comment (Svenmarim) [inline src/cryptoKeyUtil.ts]: and here

File: src/cryptoKeyUtil.ts [hunk 4]
```diff
@@ -19,13 +19,24 @@ export class CryptoKeyUtil {
...
+            privEnc: "hex",
+            pub: Buffer.from(publicKey, "hex"),
```
Comment (Svenmarim) [inline src/cryptoKeyUtil.ts]: and here

File: src/cryptoKeyUtil.ts [hunk 5]
```diff
@@ -19,13 +19,24 @@ export class CryptoKeyUtil {
...
+            pubEnc: "hex",
```
Comment (Svenmarim) [inline src/cryptoKeyUtil.ts]: etc.

**Review by Svenmarim — CHANGES_REQUESTED** [review]

**Review by Svenmarim — APPROVED** [review]

File: src/__tests__/cryptoKey.test.ts [hunk 6]
```diff
@@ -35,6 +45,27 @@ describe("Generating key pair", () => {
     });
 });
 
+describe("Generating key pair from seperate keys", () => {
+    it("Should succeed with valid public and pirvate key strings", () => {
```
Comment (timofudala) [inline src/__tests__/cryptoKey.test.ts]: Please change the typo in `private`

**Review by timofudala — CHANGES_REQUESTED** [review]

**Review by Maikkeyy — APPROVED** [review]

**Review by timofudala — APPROVED** [review]

**Pull request merged**